index.rst

```diff
@@ -0,0 +1,190 @@
+
+The images used for this test were created by GalSim simulations as described below. The galaxy profiles areies parameterized GalSim models. However, the Psfs are not from GalSim models, as we want the framework to accommodate a variety of different realistic and simulated Psfs. A separate psf image for each galaxy is used to simulate the effects of seeing:  this includes both atmospheric and optical effects. 
+
+In these tests, the Psf is modelled using SPA for later use during any measurement process where an estimation of the Psf is required (for example, for Psf weighting).
+
+Our goal is to measure enough simulated galaxies to determine the effect of different Psf approximation and shape measurement algorithms on galaxy shear estimation.
+
+DM-5447a – Techniques used in this simulations
+=================================================
+
+Creating the Psf Library:
+-------------------------
+We used PhoSim to create Psf images for an entire LSST focal plane.  The LSST focal plane is divided into 21 rafts with 9 sensors per raft.  To create a library of Psf images, 10000 positions were selected at random and PhoSim was used to create a stellar image at each position.  After removing unusable stars (those which did not fall fully on any particular sensor, or which were close to another star), between 7000 and 8000 usable stellar images remained for each simulated focal plane.
+
+Psf libraries were created for “raw seeing” values of 0.5, 0.7, and 0.9 arcseconds and for LSST filters (f2 and f3).  Most of the tests during this cycle were run using the f2_0.7 library.
+
+Using Great3Sims:
+-------------------------
+
+We used the Great3sims package to produce yaml files which in turn drive the GalSim creation of galaxy images.  Great3Sims produces a catalog of galaxies to be constructed using parameterized models and randomly drawn parameters.  A constant shear is then applied for each “subfield”, with the shear and shear angle drawn at random from a specified range.  In this respect, our galaxy sample was the same as that used for the “control” branch of the great3sims test.
+
+One important modification to Great3Sims was to allow the point spread function for each galaxy to be sampled for our Psf Library rather than from a parameterized Psf model.  The PhoSim Psf libraries described above were used to select a Psf image  GalSim during the creation of the galaxy images.  So instead of convolving the test galaxies with a parameterized Psf model, we supplied the Psf for each model as a fits image (typically 67x67).  The fits image was supplied to GalSim using the InterpolatedImage input type.
+
+The great3sims “control/ground/constant” tests provided for 200 subfields, each with a constant shear and shear angle.  There are 10000 galaxies per subfield.  Each subfield is assigned its own shear and shear angle at random.
```

Comment:
Please use consistent capitalization (we manage all of "Great3Sims", "Great3sims" and "great3sims" in the above!).

Please link to any useful external repositories or documents (e.g. links to GalSim and to Great3Sims would be nice).

Figure 1, below, claims there are an average of 9997.7 galaxies/subfield. What happened to the other 2.3?